bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigurableService.java

```diff
@@ -49,4 +49,9 @@
      */
     public static final String SERVICE_PROPERTY_CATEGORY = "service.config.category";
 
+    /**
+     * The factory pid of the service to be configured (for services with multiple contexts).
+     */
+    public static final String SERVICE_PROPERTY_FACTORY_PID = "service.factoryPid";
```

Comment:
please `org.osgi.service.cm.ConfigurationAdmin.SERVICE_FACTORYPID` instead of defining your own constant here